.github/CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 # Order is important. The last matching pattern has the most precedence.
 
-*                     @inspec/inspec-core-team
+*                     @inspec/inspec-core-team @chef/chef-infra-reviewers @chef/chef-infra-approvers @chef/chef-infra-owners
```

Comment:
@jayashrig158 as per my discussion with @clintoncwolfe  The groups that are attempted to be added here can't be added because they are not in the same github org. The @inspec github org has no communication with the @chef github org.

Reply:
@Vasu1105 sure, I will revert changes.Thanks!